documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -866,6 +866,13 @@ following sections show you how you can achieve index time boosting using Hibern
 [[section-boost-annotation]]
 ==== Static index time boosting
 
+[WARNING]
+====
+Index-time boosting is deprecated: it will be removed starting from Lucene 7.
```

Comment:
s/it will be removed starting from Lucene 7/support for this approach will be removed in Apache Lucene since version 7.